src/command_modules/azure-cli-backup/azure/cli/command_modules/backup/_params.py

```diff
@@ -76,8 +78,9 @@ def load_arguments(self, _):
         c.argument('policy', type=file_type, help='JSON encoded policy definition. Use the show command with JSON output to obtain a policy object. Modify the values using a file editor and pass the object.', completer=FilesCompleter())
 
     # Recovery Point
+    # TODO: Need to use item.id
```

Comment:
Done?

Reply:
Currently, we are using --ids to fetch the vault_name and resource_group from **vault.id**. In the future, we want to be able to fetch item_name and container_name _as well_ from --ids. Then we will use **item.id** instead of vault.id. We are not able to do this due to https://github.com/Azure/azure-cli/issues/5800